plugins/destination_test.go

```diff
@@ -0,0 +1,14 @@
+package plugins
+
+import (
+	"testing"
+
+	"github.com/cloudquery/plugin-sdk/specs"
+)
+
+func TestDestinationPlugin(t *testing.T) {
+	p := NewDestinationPlugin("test", "development", NewTestDestinationMemDBClient)
+	DestinationPluginTestSuiteRunner(t, p, specs.Destination{
```

Comment:
I'm a bit confused by this. `DestinationPluginTestSuiteRunner` runs `TestWriteOverwrite` and `TestWriteAppend`, but we set the plugin write mode to `WriteModeAppend`.

I'd expect it so each suite will run on a specific plugin write mode

Reply:
The testsuite will re-run `Init`. maybe I need to change it to get just the spec of the plugin instead of the destination.